.circleci/config.yml

```diff
@@ -294,19 +294,21 @@ jobs:
       - run: ./.circleci/brew-deploy.sh
       - slack-notify-on-failure
 
-  trigger-chocolatey-deploy:
-    docker:
-      - image: cimg/base:edge
+  chocolatey-deploy:
+    executor: windows/default
+    working_directory: chocolatey
     steps:
+      - checkout
       - run:
-          name: Trigger Chocolatey repository build
-          command: |
-            curl -X POST https://circleci.com/api/v2/project/gh/CircleCI-Public/chocolatey-circleci-cli/pipeline \
-              --fail --silent \
-              -H 'Content-Type: application/json' \
-              -H 'Accept: application/json' \
-              -H "Circle-Token: $CIRCLE_TOKEN" \
-              --data '{ "parameters": { "deploy-production": true } }'
+          name: Run update script
+          command: .\update-version.ps1
+      - run: choco pack circleci-cli\circleci-cli.nuspec
+      - run: copy-item circleci-cli.*.nupkg circleci-cli.nupkg
+      - store_artifacts:
+          path: circleci-cli.nupkg
```

Comment:
Out of curiosity, do we use this build artifact anywhere outside of the job?

Reply:
No, this is just helpful for examination - I think we can safely remove it though :)